File: week-01/day-4/src/AverageOfInput.java
import java.util.Scanner;


public class AverageOfInput {
    public static void main(String[] args) {
        // Write a program that asks for 5 integers in a row,
        // then it should print the sum and the average of these numbers like:
        //
        // Sum: 22, Average: 4.4

        Scanner scanner = new Scanner(System.in);

        System.out.println("Please give me 5 numbers!");
        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        int num3 = scanner.nextInt();
        int num4 = scanner.nextInt();
        int num5 = scanner.nextInt();

        int sum = num1 + num2 + num3 + num4 + num5;
        double avg = sum / 5.;

        System.out.println("Sum: " + sum + ", Average: " + avg);
    }
}

File: python_solutions/week-01/day-4/average_of_input.py
# Write a program that asks for 5 integers in a row,
# then it should print the sum and the average of these numbers like:
#
# Sum: 22, Average: 4.4

_sum = 0
number_of_inputs = 5

for i in range(number_of_inputs):
    _sum += int(input("number {}: ".format(i + 1)))

print("sum: {}, average: {:.1f}".format(_sum, _sum / number_of_inputs))
